File: Famoser.MassPass.Data/Entities/Communications/Response/Authorization/AuthorizationResponse.cs
using Famoser.MassPass.Data.Entities.Communications.Response.Base;

namespace Famoser.MassPass.Data.Entities.Communications.Response.Authorization
{
    public class AuthorizationResponse : ApiResponse
    {
        public string Message { get; set; }
        public string Content { get; set; }
    }
}

File: Famoser.MassPass.Api/src/Models/Response/Authorization/AuthorizationResponse.php
<?php
/**
 * Created by PhpStorm.
 * User: famoser
 * Date: 23.05.2016
 * Time: 09:40
 */

namespace Famoser\MassPass\Models\Response\Authorization;


use Famoser\MassPass\Models\Response\Base\ApiResponse;

class AuthorizationResponse extends ApiResponse
{
    public $Message;
    public $Content;
}